pandas/tests/tseries/offsets/test_common.py

```diff
@@ -133,13 +136,23 @@ def test_apply_out_of_range(request, tz_naive_fixture, _offset):
         t = Timestamp("20080101", tz=tz)
         result = t + offset
         assert isinstance(result, datetime)
+        if tz is not None:
+            assert t.tzinfo is not None
 
         if isinstance(tz, tzlocal) and not IS64 and _offset is not DateOffset:
             # If we hit OutOfBoundsDatetime on non-64 bit machines
             # we'll drop out of the try clause before the next test
             request.node.add_marker(
                 pytest.mark.xfail(reason="OverflowError inside tzlocal past 2038")
             )
+        elif (
+            isinstance(tz, tzlocal)
+            and is_platform_windows()
+            and _offset in (QuarterEnd, BQuarterBegin, BQuarterEnd)
+        ):
+            request.node.add_marker(
+                pytest.mark.xfail(reason="After GH#49737 t.tzinfo is None on CI")
+            )
```

Comment:
I'm confused -  the xfail reason only mentions `t.tzinfo`, and the if-statement condition depends on `_offset`, yet `t = Timestamp("20080101", tz=tz)` doesn't depend on `_offset`, why's that?

Reply:
best guess is some kind of windows-themed demon trying to increase my blood pressure.  seriously i couldn't reproduce it so just punted.

Reply:
OK, no objections. I don't think I've ever seen anyone use `tz_local` anyway